lib/app/index.js

```diff
@@ -135,7 +135,7 @@ async function createApp (ssrContext) {
     store[key] = app[key]
     <% } %>
     // Check if plugin not already installed
-    const installKey = '__nuxt_' + key + '_installed__'
+    const installKey = '__<%= globalName %>_' + key + '_installed__'
```

Comment:
I suggest to something like `globals.plugin` or `globals.pluginPrefix` instead of directly using `globalName`